lib/solidus_importer/processors/product_images.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class ProductImages < Base
+      def call(context)
+        @data = context[:data]
+        @entity = context[:entity]
+        context.merge!(check_data || save_images)
+      end
+
+      private
+
+      def check_data
+        if @data.blank?
+          { success: false, messages: 'Missing input data' }
```

Comment:
Can't we assume we have _some_ data here and remove this check?

Reply:
Good point 😄
I'm removing it (and the same for variant related one).

Reply:
Done, commits:
https://github.com/nebulab/solidus_importer/pull/8/commits/b2a76cc6776f5fee41fead98c412888bc363dd03
https://github.com/nebulab/solidus_importer/pull/8/commits/7a0cdd322f602d21cd697174c4ca20db39af4511